pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -72,6 +78,7 @@ func NewConfigObserver(
 				),
 			},
 			informers,
+			libgoapiserver.ObserveTLSSecurityProfile,
```

Comment:
This probably needs to be updated to accept a whitelist and potentially a path for writing the configuration to.